modules/theme/src/theme-core/definitions/apps/developer-portal.less

```diff
@@ -23,6 +23,20 @@
  @import (multiple) '../../theme.less';
  .loadVariables();
 
+/*-----------------------------
+     Google One Tap Styles
+------------------------------*/
+.google-one-tap-container {
```

Comment:
Do we use this css class in `console` and `myaccount` ?

Reply:
This should come in `login-portal.less` IMO.

Reply:
Fixed 2fb29485d8588c2690eb42a8ca6460d9854a28f0